requests/request.go

```diff
@@ -218,3 +228,44 @@ func (request *Request) Delete(path string, queryParams map[string]interface{},
 
 	return request.doRequestResponse(req)
 }
+
+func (request *Request) File(path string, params FileUploadParams) (map[string]interface{}, error) {
```

Comment:
what is this function doing can you add comments above it

Reply:
done